src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -46,6 +47,11 @@ private Drive(WPI_TalonSRX leftLead,  WPI_TalonSRX leftFollower1,  WPI_TalonSRX
 		
 		this.leftLead = leftLead;
 		initMotor(this.leftLead);
+		leftLead.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder, 0, TALON_TIMEOUT);
+		leftLead.setSensorPhase(true);
+		leftLead.config_kF(0, 0.7297, TALON_TIMEOUT); //0.7297 is 1023 (100 percent of the output you can send to the motor) divided by 1402 (max speed measured in ticks)
```

Comment:
Instead of commenting what it is, actually write in the math, and comment what it represents.